README.md

```diff
@@ -102,7 +102,7 @@ docker run \
 
 After you have installed Ape, run `ape --version` to verify the installation was successful.
 
-Interact with Ape using either the [CLI](https://docs.apeworx.io/ape/latest/index.html) or [Python code](https://docs.apeworx.io/ape/latest/index.html).
+Interact with Ape using either the [CLI](https://docs.apeworx.io/ape/latest/userguides/clis.html) or [Python code](https://docs.apeworx.io/ape/latest/userguides/scripts.html).
```

Comment:
Looking at this further, not sure if we want to point people to those userguides right off the bat

Perhaps this might look better?
```suggestion
You can interact with Ape using the [command line](https://docs.apeworx.io/ape/stable/userguides/clis.html) or the [Ape console](https://docs.apeworx.io/ape/stable/userguides/console.html).
```

Might do a follow on PR to this, thanks for the contribution!